enhancements/network/service-egress-traffic-steering.md

```diff
@@ -159,38 +169,50 @@ the egress traffic flow for the pod `10.244.1.6` on `ovn-worker2` towards an ext
+This makes the egress traffic of endpoints of an EgressService to be routed via the "blue" routing table.
+An ip rule is also created for the ClusterIP of the service which is needed in order for the ingress reply traffic (reply to an external client calling the service) to use the correct table - this is because the packet flow of contacting a LoadBalancer service goes:
 `lb ip -> node -> enter ovn with ClusterIP -> exit ovn with ClusterIP -> exit node with lb ip`
 so we need to make sure that packets from ClusterIPs are marked before being routed in order for them to hit the relevant ip rule in time.
 
+
 ### Node Selection
 Selecting a node will work similarly to how EgressIP selects one.
-When an `EgressService` resource is created for a LoadBalancer service, a node is selected to be in charge of all of its traffic.
-If the `nodeSelector` field is specified, only a node whose labels match the specified selectors can be selected.
+When an `EgressService` resource is created for a LoadBalancer service with a valid `snat` field, a node is selected to be in charge of all of its traffic.
+If the `snat.nodeSelector` field is specified, only a node whose labels match the specified selectors can be selected.
 A cache of the nodes and their number of allocations is kept in order to spread the allocations between all of the nodes available for a given service - selecting the node with the least amount of allocations every time.
 
 Once a node is selected, it is checked for readiness to serve traffic every x seconds the same way EgressIP does for its nodes (TCP/gRPC).
 If a node fails the health check, its allocations move to another node by removing the `egress-service.k8s.ovn.org/<svc-namespace>-<svc-name>: ""` label from it, removing the logical router policies from the cluster router and resetting the status of the `EgressService` which causes it to be reconciled - causing a new node to be selected for the service.
 If the node becomes not ready or its labels no longer match the service's selectors the same re-election process happens.
 
+#### Network without SNAT
+As mentioned earlier, it is possible to use the "Network" capability without SNATing the traffic to the service's ingress IP. This is done by creating an EgressService with the `Network` field specified **without** specifying the `snat` field.
+
+An EgressService without SNAT does not need to have a host selected, as the traffic will exit each node with the IP of the interface corresponding to the "Network" by leveraging the masquerade rules that are already in place.
+
+This works only on clusters running on "Local" gateway mode, because on "Shared" gateway mode the ip rules created by the controller are ignored (like all of the node's routing stack).
```

Comment:
So, `network` works in both modes when `snat` is configured right? Why can't we still create the LRP to the mgmt IP so it works without the `snat` specified for shared gw?

Reply:
replied in the other comment :)